index.js

```diff
@@ -29,6 +29,12 @@ var gulpOptions = {},
     specHtml = path.join(__dirname, '/lib/specRunner.html'),
     specRunner = path.join(__dirname, '/lib/specRunner.js');
 
+    if (!gulpOptions.phantomjsPath)
```

Comment:
This is not the correct place to put this. Instead what should be done is extending a list of default values onto `gulpOptions` down in the `module.exports`. 

What you are doing is setting a default, which is fine, but you would not be checking for existence when it is guaranteed not to exist at this point.

Also, to be future friendly you would want to introduce the idea of defaults in `gulpOptions` so that as the plugin grows and other options need default values as explicit as `'phantomjs'` and not boolean values they can be easily dropped in.